frontend/src/components/Patchnotes.jsx

```diff
@@ -56,9 +65,96 @@ const Patchnotes = ({ game }) => {
             }
         };
 
-        fetchPatchDetails();
+        fetchPatchnotes();
     }, [game, year, month, version]);
 
+    const submitComment = async (comment) => {
+        try {
+            let url = '';
+            if (game === 'overwatch') {
+                url = `http://localhost:3000/patchnotes/overwatch/${year}/${month}/comments`;
+            } else {
```

Comment:
generally, we should use `else if` with valorant and then have `else` be some kind of failure condition so that if you add more games in the future this won't break